docs/api/classes.md

```diff
@@ -7,11 +7,15 @@ The `class` resource has three possible representations: a detailed (or full) re
 ## Properties
 All properties which are not assigned with the `mandatory` label, are optional, they may not be included in the representation.
 
-* `course`: the course's unique acronym
-  - mandatory
+* `courseAcr`: the course's unique acronym
   - type: **text**
   - e.g. "WAD"
 
+* `courseId`: the course's unique identifier
+  - mandatory
+  - type: **number**
+  - e.g. 1
+
 * `calendar term`: period when the academic `event`s of the `course` will occur
   - mandatory
   - type: **text**
```

Comment:
Same thing as the `ClassSection` relation comment

Reply:
API interactions are independent from the physical model. User interaction don't have to change even if the underlying database changes.